pkg/converters/types/options.go

```diff
@@ -22,25 +22,26 @@ import (
 
 // ConverterOptions ...
 type ConverterOptions struct {
-	Logger           types.Logger
-	Cache            Cache
-	Tracker          Tracker
-	DynamicConfig    *DynamicConfig
-	LocalFSPrefix    string
-	IsExternal       bool
-	MasterSocket     string
-	AdminSocket      string
-	AcmeSocket       string
-	DefaultConfig    func() map[string]string
-	DefaultBackend   string
-	DefaultCrtSecret string
-	FakeCrtFile      CrtFile
-	FakeCAFile       CrtFile
-	AnnotationPrefix []string
-	DisableKeywords  []string
-	AcmeTrackTLSAnn  bool
-	TrackInstances   bool
-	HasGateway       bool
+	Logger                  types.Logger
+	Cache                   Cache
+	Tracker                 Tracker
+	DynamicConfig           *DynamicConfig
+	LocalFSPrefix           string
+	IsExternal              bool
+	MasterSocket            string
+	AdminSocket             string
+	AcmeSocket              string
+	DefaultConfig           func() map[string]string
+	DefaultBackend          string
+	DefaultCrtSecret        string
+	FakeCrtFile             CrtFile
+	FakeCAFile              CrtFile
+	AnnotationPrefix        []string
+	DisableKeywords         []string
+	AcmeTrackTLSAnn         bool
+	TrackInstances          bool
+	HasGateway              bool
+	EnableEndpointSlicesAPI bool
```

Comment:
This wasn't initialized, here:

https://github.com/jcmoraisjr/haproxy-ingress/blob/b87f94b9149bad3cef902e7e058c62ad2d83e02e/pkg/controller/controller.go#L156-L175